packages/ai/cypress/specs/Input.cy.tsx

```diff
@@ -0,0 +1,950 @@
+import AIInput from "../../src/AIInput.js";
+import MenuItem from "@ui5/webcomponents/dist/MenuItem.js";
+import "@ui5/webcomponents-icons/dist/ai.js";
+import "@ui5/webcomponents-icons/dist/stop.js";
+import { WRITING_ASSISTANT_LABEL } from "../../src/generated/i18n/i18n-defaults.js";
+
+describe("Basic", () => {
+    describe("Initialization", () => {
+        it("should render with default properties", () => {
+            cy.mount(<AIInput />);

```

Comment:
When I render an AI Input without any menu items in the slot, the AI icon is present and when I click on it - an empty menu item with no text appears. Maybe it is better to not render ghost menu item in that case